README.md

```diff
@@ -28,6 +28,10 @@
 - Twitter: [@meshu102](https://twitter.com/meshu102)
 - LinkedIn:  [Linkedin](https://www.linkedin.com/in/melashu-amare-033a50b8/)
 
+## Live Demo 
+
+[My Potfolio](https://melashu.github.io/My-Portfolio/)
+
```

Comment:
- Good job implementing and deploying your portfolio project, But there are some issues as your navigation link does not navigate to the appropriate section when clicked, Please try to fix it.

Reply:
@Ginohmk  I Just wanna say thank you for your constructive feedback. But I didn't really understand the issue. When I click the Live Demo link it navigate me to the page. Can you please give more hint about that?